README.md

```diff
@@ -35,29 +35,39 @@ The GitHub Actions context has access to a `GITHUB_TOKEN` environment variables
 - To learn more on token scopes [click here](https://developer.github.com/apps/building-oauth-apps/understanding-scopes-for-oauth-apps/#available-scopes).
 - To learn how to create your own personal access token [click here](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line).
 
-### Parsing Rules
+### Config Rules
 
-A JSON file with the rules you need to define to parse the GitHub Issue body and extract the data needed to create an invitation to your GitHub organization.
+A JSON file with the rules you need to define to parse the GitHub Issue body and extract the data needed to create an invitation to your GitHub organization as well as the valid domain from which you will accept emails.
 
 #### Structure
 
-The action expects the use of regular expressions with named capture groups. There are two base named capture groups that the Action expects:
+The action expects the use of regular expressions with named capture groups. There are two base named capture groups that the Action expects with one additional optional group:
 
-- **username**
-- **email**
+- **emailRule**
+- **parsingRules**
+- **userCreatedRule**
+  - Optional, validation will be ignored if this is not included
 
 ```JSON
 {
-  "username": {
-    "regex": "your-regular-expression (?<username>.+?)"
+  "emailRule": {
```

Comment:
Since we want validate email domain I think this should be called

```suggestion
  "emailDomainRule": {
```

Reply:
going to accept this since this is going to be our standard use case, but it can be a regex so consumers of this action could do a lot of different things depending on their needs